### PR TITLE
refactor: make @remax/types more like types

### PR DESCRIPTION
**Pull request by hehex9**
should update typescript template after merged since `@remax/types` has been moved to devDependencies now.

### CODE REVIEW

File: packages/babel-plugin-remax-host-component/tests/index.test.ts [hunk 1]
```diff
@@ -15,7 +14,7 @@ function transform(code: string, filename?: string) {
         plugins: [
           '@babel/plugin-syntax-jsx',
           component({
-            target: Platform.ali,
+            target: 'ali',
```
Comment (yesmeck) [inline packages/babel-plugin-remax-host-component/tests/index.test.ts:17]: 这个 Enum 要不放到 shared 里？要写字符串不是很方便。

Comment (hehex9) [reply packages/babel-plugin-remax-host-component/tests/index.test.ts:17]: 毕竟还有代码提示，应该 [还](https://github.com/apollographql/apollo-client/blob/main/src/core/watchQueryOptions.ts#L18) [还](https://github.com/apache/incubator-echarts/blob/master/src/util/types.ts#L59) [还](https://github.com/facebook/jest/blob/master/packages/jest-types/src/Config.ts#L232) [还](https://github.com/expo/expo/blob/master/packages/%40unimodules/react-native-adapter/src/Platform.ts#L5) 好吧？ 🤭

**Comment by yesmeck** [comment]
> should update typescript template after merged since @remax/types has been moved to devDependencies now.

模板需要改什么呢？

**Comment by hehex9** [comment]
ts 模板需要加上 @remax/types 作为 devDependencies

**Comment by yesmeck** [comment]
`@remax/types` 可以作为 `remax` 的 dependencies 吧？

**Pull request merged**